internal/packets/login.go

```diff
@@ -68,6 +68,8 @@ const (
 	ShipSelection
 )
 
+const ShipSelectionPhase = 0xE04
```

Comment:
As is, this will break my client. Maybe remove this from this PR until we've figured out what's up with this login phase stuff?

Reply:
Ok. But how about shortening the field e.g. splitting Phase into:
```
Phase         uint8
Unknown4      uint8 // or any other name you propose
```

Reply:
Let's try that based on what I put in Discord, can you toss a comment above when you do that and say that these might be related fields?

Reply:
Done